src/digitalmarketplace/components/cookie-settings/cookie-settings.js

```diff
@@ -99,13 +99,15 @@ CookieSettings.prototype.showErrorMessage = function () {
     errorMessage.style.display = 'block'
     document.body.scrollTop = document.documentElement.scrollTop = 0
   }
+  showInlineErrorMessage()
 }
 
 CookieSettings.prototype.hideErrorMessage = function () {
   var errorMessage = document.querySelector('#dm-cookie-settings-error')
   if (errorMessage !== null) {
     errorMessage.style.display = 'none'
   }
+  hideInlineErrorMessage()
```

Comment:
To help with testing, I'd probably have these as separate prototype methods, rather than nesting inside the existing ones. Feel free to rename the existing methods if it makes it clearer:

```
CookieSettings.prototype.showErrorSummary = function () {
   ...
}

CookieSettings.prototype.hideErrorSummary = function () {
   ...
}

CookieSettings.prototype.showInlineErrorMessage = function () {
   ...
}

CookieSettings.prototype.hideInlineErrorMessage = function () {
   ...
}

```